example/src/App.tsx

```diff
@@ -154,6 +157,19 @@ const App = () => {
     setExpanded(batchesTreeViewExpandedIds);
   };
 
+  const deInitData = ethers.utils.defaultAbiCoder.encode(
+    ["address", "bytes"],
+    ['0x0000000000000000000000000000000000000001', '0x00']
+  );
+
+  const onInstallModuleClick = async () => {
+    await installModule(MODULE_TYPE.VALIDATOR, '0x6a00da4DEEf677Ad854B7c14F17Ed9312c2B5fDf');
```

Comment:
It's worth annotating here what module this is for other developers

Reply:
Yes sure, it is a test module on Sepolia, I am adding this